src/App.tsx

```diff
@@ -1,61 +1,119 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { FormData, Todo } from './types';
+import { TodoList } from './components/TodoList';
+import { getNewTodoId } from './services';
+
+const initialForm: FormData = {
+  title: '',
+  userId: 0,
+};
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
```

Comment:
The initial state for `todos` is set using `todosFromServer`. Ensure that this data is correctly fetched and structured as expected.